staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5635,7 +5635,7 @@ type ServiceProxyOptions struct {
 // Instead of using this type, create a locally provided and used type that is well-focused on your reference.
 // For example, ServiceReferences for admission registration: https://github.com/kubernetes/api/blob/release-1.17/admissionregistration/v1/types.go#L533 .
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-// +structType=atomic
+// +structType=granular
```

Comment:
Do we want to keep it called out or should we just remove the structType marker altogether?